orchagent/portsorch.cpp

```diff
@@ -4084,9 +4084,19 @@ void PortsOrch::initializePortMaximumHeadroom(Port &port)
         return;
     }
 
-    vector<FieldValueTuple> fvVector;
     port.m_maximum_headroom = attr.value.u32;
-    fvVector.emplace_back("max_headroom_size", to_string(port.m_maximum_headroom));
+}
+
+void PortsOrch::initializePortBufferMaximumParameters(Port &port)
```

Comment:
can this be combined with initializePortMaximumHeadroom?

Reply:
ok. let's change the name to `initializePortBufferMaximumParameters` and then merge them together.

Reply:
Fixed